content/id/docs/concepts/cluster-administration/federation.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Federation
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+
+{{< deprecationfilewarning >}}
+{{< include "federation-deprecation-warning-note.md" >}}
+{{< /deprecationfilewarning >}}
+
+Laman ini menjelaskan alasan dan cara penggunaan _federation_ untuk melakukan manajemen 
+kluster Kubernetes.
+{{% /capture %}}
+
+{{% capture body %}}
+## Kenapa _Federation_ ?
+
+_Federation_ membuat proses manajemen _multiple_ kluster menjadi lebih mudah. 
+_Federation_ mencapai hal ini dengan cara menyediakan 2 buah fondasi:
+
+  * Melakukan sinkronisasi _resource_ di seluruh kluster: _Federation_ 
+    menyediakan kemampuan untuk melakukan sinkronisasi _resources_ pada _multiple_ 
+    kluster. Sebagai contoh, kamu dapat memastikan `Deployment` yang sama 
+    tersedia pada _multiple_ kluster. 
+  * _Cross_ _cluster_ _discovery_: _Federation_ menyediakan kemampuan untuk melakukan 
+     konfigurasi otomatis server DNS dan _load balancer_ dari semua kluster. 
+     Misalnya, kamu dapat memastikan bahwa sebuah VIP atau DNS global dapat digunakan 
+     untuk mengakses _backend_ dari _multiple_ kluster.
+
+Beberapa penggunaan _federation_ adalah sebagai berikut:
+
+* _High Availability_: Melakukan _load balance_ di seluruh kluster serta 
+  melakukan konfigurasi otomatis server DNS dan _load balancer_, _federation_ 
+  meminimalisasi dampak yang terjadi apabila terjadi kegagalan kluster.
+* Mencegah _lock-in_ yang terjadi akibat penyedia layanan: Dengan cara mempermudah 
+  proses migrasi antar kluster.
+
+
+Manfaat _federation_ tidak akan terlalu kelihatan kecuali kamu memiliki beberapa kluster. 
+Beberapa alasan kenapa kamu butuh beberapa kluster adalah:
+
+* _Latency_ yang rendah: Memiliki kluster yang berada di _region_ yang berbeda 
+  meminimalisasi _latency_ dengan cara menyajikan konten ke pengguna 
+  berdasarkan _region_ yang paling dekat dengan pengguna tersebut.
+* Isolasi _fault_: Akan lebih baik apabila kita memiliki beberapa kluster kecil 
+  dibandingkan sebuah kluster besar untuk melakukan isolasi _fault_ (misalnya saja 
+  kluster ini bisa saja berada di _availability_ _zone_ dan penyedia layanan _cloud_ 
+  yang berbeda).
+* _Scalability_: Terdapat batasan _scalability_ untuk sebuah kluster Kubernetes, 
```

Comment:
use Skalabilitas?

https://id.wikipedia.org/wiki/Skalabilitas

although not standardized yet in KBBI, wiki gives us choices: skalabilitas or keterluasan